ape_foundry/provider.py

```diff
@@ -207,6 +207,16 @@ def uri(self) -> str:
             self._host = f"http://127.0.0.1:{DEFAULT_PORT}"
         return self._host
 
+    @property
+    def http_uri(self) -> str:
+        # NOTE: Overriding `Web3Provider.http_uri` implementation
+        return self.uri
+
+    @property
+    def ws_uri(self) -> str:
```

Comment:
is this URL correct? I can't seem to connect it or see anything I need to do to enable WS

Reply:
seems to work for me :shrug: